[3주차] CSS

kingmingseo opened the pull request:
## ✅ 학습 내용 요약

- 어떤 학습을 했는지 간단하게 요약해주세요.

## 🤔 궁금했던 부분

- 학습중 궁금했던 부분을 적어주세요.
- 궁금했던 부분을 해결하셨다면 **어떤 과정**을 통해 해결하셨는지 작성해주세요.
- 경험 공유나 다른 팀원들의 생각을 들을 수 있을 것 같아요.

kingmingseo committed:
Create 김민서.md

kingmingseo committed:
[3주차] 김민서

kingmingseo commented:
.

yougyung commented:
[2주차] 입문자를 위한 HTML 기초 강의과 학습내용이 일치하지않네요. 올려주신 목록 확인해주세요.

kingmingseo commented:
> [2주차] 입문자를 위한 HTML 기초 강의과 학습내용이 일치하지않네요. 올려주신 목록 확인해주세요.

제가 1주차 강의가 너무 금방 끝나서 커리큘럼에 나온 
1주차(9.20~ 9.26)
주차별 학습: HTML 강의(목록 표시하기까지)
2주차(9.27 ~ 10.3)
주차별 학습: HTML 강의(뷰포트)

위 의 2주간 양의 강의를 1 주차 때 모두 수강 후 1주차 때 PR 하고 

2주차 때 
[입문자를 위한 CSS 기초 강의] 를 수강 하였으며 

3주차 때 
[CSS 레이아웃 FLEX -GRID]를 수강해서 순서가 조금 엇갈린 것 같습니다.

제목을 어떻게 하면 좋을까요 ?

yougyung commented:
> > [2주차] 입문자를 위한 HTML 기초 강의과 학습내용이 일치하지않네요. 올려주신 목록 확인해주세요.
> 
> 제가 1주차 강의가 너무 금방 끝나서 커리큘럼에 나온 1주차(9.20~ 9.26) 주차별 학습: HTML 강의(목록 표시하기까지) 2주차(9.27 ~ 10.3) 주차별 학습: HTML 강의(뷰포트)
> 
> 위 의 2주간 양의 강의를 1 주차 때 모두 수강 후 1주차 때 PR 하고
> 
> 2주차 때 [입문자를 위한 CSS 기초 강의] 를 수강 하였으며
> 
> 3주차 때 [CSS 레이아웃 FLEX -GRID]를 수강해서 순서가 조금 엇갈린 것 같습니다.
> 
> 제목을 어떻게 하면 좋을까요 ?

1,2주차의 HTML 학습내용을 1주차에 올려주셨고, 해당 문서는 3주차의 강의를 올려주신 것으로 이해했습니다.
[1,2 주차] 입문자를 위한 HTML 기초 강의
[3주차] CSS
로 변경해둘테니, 혹시 민서님께서 PR제목이 학습내용에 적절하지 않은 것 같다는 생각이 드시면 학습한 내용과 일치하는 적절한 제목으로 변경해서 올려주시면 좋을 듯 합니다. 
매주 주차에 해당하는 학습 범위를 올려주셔합니다. 따라서 학습 범위에 따라 문서를 구분해서 정리해두시길 권장드려요. 이번 상황처럼 예외적으로 설명해주실 부분이 있다면 pr에 함께 작성해주시면 더 좋을것같아요. 한 주도 수고하셨어요  :) 👍

yougyung commented on week03/CSS 레이아웃 flex- grid a0a110bcf5b341299b9ac483f5155442.md:
```diff
@@ -0,0 +1,333 @@
+
+grid-column과 grid-low 속성을 이용하면 그리드 컨테이너의 줄 번호를 이용해 아이템을 배치할 수 있다.
+
+![Untitled](CSS%20%E1%84%85%E1%85%A6%E1%84%8B%E1%85%B5%E1%84%8B%E1%85%A1%E1%84%8B%E1%85%AE%E1%86%BA%20flex-%20grid%20a0a110bcf5b341299b9ac483f5155442/Untitled%201.png)
```
사진 파일 적용이 안된 것 같아요!!
https://adjh54.tistory.com/100 참고해서 수정해주세요~

yougyung commented on 김민서.md:
```diff
@@ -0,0 +1,632 @@
+
+선택자 우선순위는
+
+**id선택자 > 클래스 선택자 > 태그선택자** 
```
인라인 스타일과 !important 포함한 우선순위도 확인해주세요

yougyung commented on 김민서.md:
```diff
@@ -0,0 +1,632 @@
+
+z-index 값의 기본 값은 auto이다. auto는 새로운 쌓임 맥락이 형성되지 않은 자연스러운 상태이다.
+
+# 하나 더 알게된 사항
```
👍 👍

kingmingseo committed:
[4주차] 김민서

정해주신 진도까지의 학습을 마치지 못했습니다.. 
시험기간의 활동을 하지 않는 기간에 마저 학습하여 진도 맞추고 팀에 피해가 가지 않게 하겠습니다.

kingmingseo committed:
Merge branch 'COW-edu:main' into main

kingmingseo committed:
Create test

kingmingseo committed:
Create test.md

kingmingseo committed:
Delete 7주차 directory

kingmingseo committed:
Create test.md

kingmingseo committed:
Add files via upload

kingmingseo committed:
[7주차] 김민서
